examples/src/app/example.tsx

```diff
@@ -1,15 +1,29 @@
 import React, { Component } from 'react';
-import { Container, Spinner } from '@playcanvas/pcui/react';
+import { Container, Spinner, SelectInput, Panel } from '@playcanvas/pcui/react';
+import { SelectInput as SelectInputClass } from '@playcanvas/pcui';
 import { File } from './helpers/types';
 import examples from './helpers/example-data.mjs';
 // @ts-ignore: library file import
 import { withRouter } from 'react-router-dom';
 import ControlPanel from './control-panel';
+import { Observer } from '@playcanvas/observer';
+
+const DEVICETYPE_WEBGL1 = 'webgl1';
```

Comment:
do we need these here or can they come from engine constants

Reply:
I've updated these to come from the engine build.